resources/views/vendor/laravel-log-viewer/smtp/form.blade.php

```diff
@@ -0,0 +1,59 @@
+<div class="form-group">
+    <label class="control-label col-md-3 col-sm-3 col-xs-12" for="provider">Penyedia Server</label>
+
+    <div class="col-md-6 col-sm-6 col-xs-12">
+        {!! Form::select('provider', ['smtp' => 'SMTP', 'webmail' => 'Webmail', 'google' => 'Google'], $email_smtp->provider, ['class' => 'form-control']) !!}
```

Comment:
Error ketika sudah ada tabelnya namun belum ada record smtp tersebut

[previous exception] [object] (ErrorException(code: 0): Attempt to read property \"provider\" on null at /data/docker/opendesa/OpenDK/storage/framework/views/17245156f4a55f731b523300f0b60620db8afa3c.php:5)
[stacktrace]

![image](https://github.com/user-attachments/assets/85c43c55-35cb-4aa6-8cdf-55220ad031db)